detect/nulldetector.py

```diff
@@ -28,7 +29,10 @@ def detect_noisy_cells(self):
         attributes = self.ds.get_attributes()
         errors = []
         for attr in attributes:
-            tmp_df = self.df[self.df[attr] == '_nan_']['_tid_'].to_frame()
+            # do not add attributes in which all values are null
+            if (self.df[attr] == NULL_REPR).all():
```

Comment:
Why did we add this again? Don't we drop all-null columns?

Reply:
Yes, I think the branch I checked out was based on an older version of dev so the change with `NULL_REPR` introduced it back in during the conflict. Removed.